output.go

```diff
@@ -3,20 +3,58 @@ package jasper
 import (
 	"io"
 	"io/ioutil"
+	"os"
 
 	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/send"
 	"github.com/pkg/errors"
 )
 
 // OutputOptions provides a common way to define and represent the
 // output behavior of a evergreen/subprocess.Command operation.
 type OutputOptions struct {
-	Output            io.Writer `json:"-"`
-	Error             io.Writer `json:"-"`
-	SuppressOutput    bool      `json:"suppress_output"`
-	SuppressError     bool      `json:"suppress_error"`
-	SendOutputToError bool      `json:"redirect_output_to_error"`
-	SendErrorToOutput bool      `json:"redirect_error_to_output"`
+	Output            io.Writer          `json:"-"`
+	Error             io.Writer          `json:"-"`
+	SuppressOutput    bool               `json:"suppress_output"`
+	SuppressError     bool               `json:"suppress_error"`
+	SendOutputToError bool               `json:"redirect_output_to_error"`
+	SendErrorToOutput bool               `json:"redirect_error_to_output"`
+	Loggers           []Logger           `json:"loggers"`
+	OutputSender      *send.WriterSender `json:"-"`
+	ErrorSender       *send.WriterSender `json:"-"`
+}
+
+type LogType string
+
+const (
+	LogBuildloggerV2 = "buildloggerv2"
+	LogBuildloggerV3 = "buildloggerv3"
+	LogDefault       = "default"
+	LogFile          = "file"
+	LogInherit       = "inherit"
+	LogSplunk        = "splunk"
+	LogSumologic     = "sumologic"
+)
+
+const (
+	DefaultLogName = "jasper"
+)
+
+type LogOptions struct {
+	BuildloggerOptions send.BuildloggerConfig    `json:"buildlogger_options"`
+	DefaultPrefix      string                    `json:"default_prefix"`
+	FileName           string                    `json:"file_name"`
+	SplunkOptions      send.SplunkConnectionInfo `json:"splunk_options"`
+	SumoEndpoint       string                    `json:"sumo_endpoint"`
+	// By default, logger reads from both standard output and standard error.
+	IgnoreOutput bool `json:"ignore_log_output"`
+	IgnoreError  bool `json:"ignore_log_error"`
```

Comment:
I think that this should just respect the OutputOptions values.

Reply:
Done.